src/edu/csus/ecs/pc2/core/InternalController.java

```diff
@@ -3251,19 +3196,28 @@ private Profile getCurrentProfile() {
 
     private void handleCommandLineOptions() {
 
-        if (parseArguments.isOptPresent("--help")) {
+        if (parseArguments.isOptPresent(AppConstants.HELP_OPTION_STRING)) {
             // -F is the ParseArguements internal option to pre-load command line options from a file
 
             String[] usage = { //
-                    "Usage: Starter [--help] [-F filename] [--server] [--first] [--login <login>] [--password <pass>] " + //
-                    "[" + LOAD_OPTION_STRING + " <dir>|<file> ] " + //
-                    "[--skipini] " + //
-                    "[" + NOLOGGING_OPTION_STRING + "] ", //
-                    "[" + INI_FILENAME_OPTION_STRING + " filename] [" + //
-                    CONTEST_PASSWORD_OPTION + " <pass>] [" + NO_GUI_OPTION_STRING + "] "+ //
-                    "[" + NOSTANDINGS_OPTION_STRING + "] "+ //
-                    "[" + REMOTE_SERVER_OPTION_STRING + " <remoteHostname> --proxyme]" + //
-                    "[" + MAIN_UI_OPTION + " classname]", // 
+                    "Usage: Starter " + //
+                    "[" + AppConstants.HELP_OPTION_STRING + "] " + //
+                    "[" + AppConstants.FILE_OPTION_STRING + " filename] " + //
+                    "[" + AppConstants.SERVER_OPTION_STRING + "] " + //
+                    "[" + AppConstants.FIRST_SERVER_OPTION_STRING + "] " + //
+                    "[" + AppConstants.LOGIN_OPTION_STRING + " <login> " + //
+                    "[" + AppConstants.PASSWORD_OPTION_STRING + " <pass> ", //
+                    "           " + 
+                    "[" + AppConstants.LOAD_OPTION_STRING + " <dir>|<file> ] " + //
+                    "[" + AppConstants.SKIP_INI_OPTION_STRING + "] " + //
+                    "[" + AppConstants.INI_FILENAME_OPTION_STRING + " INIfilename] " + //
+                    "[" + AppConstants.CONTEST_PASSWORD_OPTION_STRING + " <pass>] " + 
+                    "[" + AppConstants.REMOTE_SERVER_OPTION_STRING + " <remoteHostname> " + AppConstants.PROXY_ME_SERVER_KEY + "]" + //
+                    "[" + AppConstants.MAIN_UI_OPTION_STRING + " classname]", // 
+                    "           " +
+                    "[" + AppConstants.NOLOGGING_OPTION_STRING + "] " + //
+                    "[" + AppConstants.NO_GUI_OPTION_STRING + "] " + //
+                    "[" + AppConstants.NOSTANDINGS_OPTION_STRING + "] ", //
                     "", //
                     "See http://pc2.ecs.csus.edu/wiki/Command_Line for more information", // 
```

Comment:
This line should be replaced with `https://github.com/pc2ccs/pc2v9/wiki/Command-Line-Arguments`   (I realize it's not part of this Issue/PR, but it's good to replace bad references wherever they pop up...)

Reply:
Will be updated in the next push.